### PR TITLE
JM: Added matches and players adding/deleting/modifying

### PR DESCRIPTION
**Pull request by NitroBolon**
All done (hopefully)

### CODE REVIEW

File: SoccerPlanner/SoccerPlanner/urls.py [hunk 1]
```diff
@@ -43,4 +45,4 @@
     path('stagedeletesuccessful',views.stagedeletesuccessful, name = 'stagedeletesuccessful'),
     path('captcha', views.captcha,name='captcha'),
     path('stageeditsuccessful',views.stageeditsuccessful, name='stageeditsuccessful')
-]
```
Comment (guestnone) [inline SoccerPlanner/SoccerPlanner/urls.py]: This should not be removed, since this is an array

File: SoccerPlanner/app/forms.py [hunk 2]
```diff
@@ -27,6 +28,62 @@ class Meta:
...
+
+class PlayerDeleteForm(ModelForm):
+    listOfPlayers = forms.ModelChoiceField(queryset = Player.objects.all(), label = "Player", required = True, empty_label ="(None)")
+    class Meta:
+        model = Player
+        fields = ('listOfPlayers',)
+
+class MatchDeleteForm(ModelForm):
+    listOfMatches = forms.ModelChoiceField(queryset = Match.objects.all(), label = "Match", required = True, empty_label ="(None)")
+    class Meta:
```
Comment (guestnone) [inline SoccerPlanner/app/forms.py:78]: If class Meta is empty, VS screams about indentation. This should be removed.

**Review by guestnone — COMMENTED** [review]
I'm currently trying to launch your PR, so I'm going to comment on the required changes (I  will leave them on proper lines).
   * django-recapcha2 library needs to be added to the requirements.txt (to install dependencies), but since this is not on your side I will do it.
   * Python screams about those errors:
app.Stage_listOfMatches: (fields.E336) The model is used as an intermediate model by 'app.Stage.listOfMatches', but it does not have a foreign key to 'Stage' or 'Match'.
app.Tournament.stage: (fields.E300) Field defines a relation with model 'Stage', which is either not installed, or is abstract.

File: SoccerPlanner/app/forms.py (hunk 2 above)
Comment (NitroBolon) [reply SoccerPlanner/app/forms.py:78]: There should be something like 
"model = Match
fields = ('listOfMatches',)"
in Meta part, idk why it`s empty :/

Comment (NitroBolon) [reply SoccerPlanner/app/forms.py:78]: As I see, MatchDeleteForm`s Meta is not empty anymore
loll

File: SoccerPlanner/SoccerPlanner/urls.py (hunk 1 above)
Comment (NitroBolon) [reply SoccerPlanner/SoccerPlanner/urls.py]: Ok, done.

**Pull request closed without merging**